DOCKER/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [[ $BARAK_SEED ]]; then
+  cat /go/src/github.com/eris-ltd/eris-db/Godeps/_workspace/src/github.com/tendermint/tendermint/cmd/barak/$BARAK_SEED | barak &
```

Comment:
this doesn't work since tendermint/tendermint/cmd is not a dependency (its not vendored - godeps is at least some what intelligent)